.github/workflows/generate-binaries.yaml

```diff
@@ -44,14 +44,18 @@ jobs:
           echo "$PWD" >> $GITHUB_PATH
 
       - name: Build binaries
-        run: cargo build --features=ispc
+        run: |
+          cargo build --features=ispc --target=x86_64-apple-darwin
+          cargo build --features=ispc --target=aarch64-apple-darwin
```

Comment:
Just like d8c2a1e4b244b523b6282122ba562b86b3be1117 you probably have to add a `rustup target add ...` step above.